website/moneybirdsynchronization/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from thaliawebsite import settings
+
+from members.models import Member
+
+class Contact(models.Model):
```

Comment:
I think our CI requires you to override __str__ and define a meta class with verbose names